article.py

```diff
@@ -1,9 +1,10 @@
 """
```

Comment:
discard changes in this file

Reply:
fixed it

Reply:
create a separate PR with the change that fix the lint and @dmitry-uraev will review and merge if accepted. however, in this PR you have to discard the changes